pkg/search/postgres/store.go

```diff
@@ -23,7 +23,7 @@ import (
 const (
 	batchAfter      = 100
 	cursorBatchSize = 50
-	deleteBatchSize = 5000
+	deleteBatchSize = 65000
```

Comment:
just for my own sake: any particular story behind the number other than reducing the amount of transactions we create?

Reply:
Yeah, let me take this number and make it a global var. Basically, there is a max number of params that postgres can handle in a query (65,535)